### ISSUE TITLE
"Owned by Microsoft"

### ISSUE THREAD

**Issue by phrequencyviii** [nikitavoloboev/privacy-respecting#58]
I love this list, but I notice the only reason Microsoft's products are in the "bad" list is "owned by Microsoft". They aren't really too bad nowadays on privacy, especially compared to Google, or how they used to be. It just seems a little biased to me, especially since Apple Siri is included in the good list, but they were using contractors to listen to audio snippets without being clear, etc.

**Comment by nikitavoloboev** [comment]
I would agree. Can accept PR with changes you like to see that are more fair.

**Comment by ph00lt0** [comment]
> They aren't really too bad nowadays on privacy

They are part of the PRISM program if I am correct. Apple's siri is also not recommendable but it's likely a lot better then Cortana which is used for analytics from Bing.
